### ISSUE TITLE
typescript-estree is not supporting the 3.4v typescript

### ISSUE THREAD

**Issue by denniscual** [typescript-eslint/typescript-eslint#393]
After installing the latest version of typescript 3.4, I got this error.

`You are currently running a version of TypeScript which is not officially supported by typescript-estree.`

Can you help me how to fix this error wherein I can try to use the latest version of typescript in my project without this error?

**Comment by thebanjomatic** [comment]
The project's readme contains instructions for how to suppress this warning: 
https://github.com/typescript-eslint/typescript-eslint/blob/master/README.md#supported-typescript-version

That being said, I'm interested in knowing when 3.4 will be officially supported. It would be nice to see same-day launch support in the future given the beta's are out for a couple weeks prior to typescript releasing.

**Comment by JamesHenry** [comment]
@thebanjomatic 

> It would be nice to see same-day launch support in the future given the beta's are out for a couple weeks prior to typescript releasing.

Please don't forget this is open-source, run by volunteers in their spare time, and contributions are always welcome!

**Comment by JamesHenry** [comment]
This was released in `1.6.0`

**Comment by OnkelTem** [comment]
@thebanjomatic 

> The project's readme contains instructions for how to suppress this warning:
> https://github.com/typescript-eslint/typescript-eslint/blob/master/README.md#supported-typescript-version

Sorry, but I couldn't supress this warning anyway.
I created `.eslintrc.json` file in the root, add there lines:

```json
{
  "parserOptions": {
    "warnOnUnsupportedTypeScriptVersion": false
  }
}
```

but it didn't help at all. Any ideas how to supress the warning?

**Comment by bradzacher** [comment]
have a look into yarn resolutions to help with resolving multiple packages requiring different versions of packages.
https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

Also looking at your post - create-react-app does its own thing with eslint which means your parserOptions probably aren't being passed in. This is outside our control, and I recommend you hop over to their repo to track the status of their efforts to allow configurable eslint.
